CRM/Upgrade/Incremental/SmartGroups.php

```diff
@@ -71,6 +71,8 @@ public function datePickerConversion($fields) {
       'case_start_date' => 'case_from',
       'case_end_date' => 'case_to',
       'mailing_job_start_date' => 'mailing_date',
+      'relationship_start_date' => 'relation_start_date',
```

Comment:
I think the old name was just relation_start i think and relation_end for the other i think

Reply:
looks like relation_start_date here https://github.com/civicrm/civicrm-core/pull/15637/files#diff-54576153cb6a72d6ea7e22a5e8ca63f2L406

Reply:
odd one because when i created from an advanced search the form values was

```
s:14:"relative_dates";a:1:{s:14:"relation_start";s:10:"this.month";}
```

Reply:
ug

Reply:
well I guess that is authoritive if you did that test

Reply:
@seamuslee001 actually the UI ALSO has the field relation_date - which I hadn't made sense of - maybe it's the equivalent of the event_date? Although my initial assumption (ie it's just broken) may also be true & the resulting smart group from the pre-this change search might not actually work .... I guess we need to test that

Reply:
@eileenmcnaughton i can't even see where that relation_date date search is exposed on the form its self, i think just get rid of it

Reply:
looking at this @eileenmcnaughton 
https://github.com/civicrm/civicrm-core/pull/15637/files#diff-54576153cb6a72d6ea7e22a5e8ca63f2R410 it would suggest it only goes until it hits either _date or _relative which i think matches with it storing relation_start in the db